### ISSUE TITLE
Does ContentReader of DownloadFileswithsummary have file information in Bytes

### ISSUE THREAD

**Issue by I559856** [jfrog/jfrog-client-go#705]
Is there any way to get file in bytes from Downloadwithsummary ? I see contentReader returns only returns URI's in datachannels .

**Comment by I559856** [comment]
Hi team, is there any update on this issue ?

**Comment by Or-Geva** [comment]
Hello @I559856. Following this [documentation](https://github.com/jfrog/jfrog-client-go#uploading-and-downloading-files-with-summary), you can get the file path, read it, and get the file data as bytes.
If this does not help you, please share more information about your end goal.

**Comment by I559856** [comment]
We wanted to avoid additional IO operation. Jfrog Rest API returns byte stream similarly I wanted to know if content reader returns any bytes directly.

**Comment by Or-Geva** [comment]
Got it @I559856 , the current API does not support streams, would you like to contribute code to jfrog-client-go?
To get started, [here](https://github.com/jfrog/jfrog-client-go#reading-files-in-artifactory) is an example of a similar API that returns a stream of the file content (without a summary).

**Comment by I559856** [comment]
Yes , I would love to, I am going through the code will shortly raise an PR
